src/ol_openedx_chat/block.py

```diff
@@ -36,22 +40,70 @@ class OLChatAside(XBlockAside):
     XBlock aside that enables OL AI Chat functionality for an XBlock
     """
 
-    @XBlockAside.aside_for("student_view")
-    def student_view_aside(self, block, context=None):  # noqa: ARG002
+    enabled = Boolean(
+        display_name=_("Open Learning Chat enabled status"),
+        default=False,
+        scope=Scope.content,
+        help=_("Indicates whether or not Open Learning chat is enabled for a block"),
+    )
+    chat_prompts = String(
+        display_name=_("Open Learning Chat Prompt text"),
+        default="",
+        scope=Scope.content,
+        help=_("Prompt hint text for chat in a block"),
+    )
```

Comment:
How are we going to handle multiple prompts? Comma separated maybe OR some other separator?